README.md

```diff
@@ -109,29 +104,22 @@ That is how we avoid
 storing keys in the container itself.
 ```
 docker network create cpud
-# If you ran docker before, you need to remove the
-# old identity.
-docker rm cpud_test
-docker run --mount type=bind,source=$KEY.pub,target=/key.pub --mount type=bind,source=$KEY,target=/key --name cpud_test --privileged=true -t -i -p 17010:17010 rminnich/cpu:latest
-```
-
-Note: once the container is done, if you want to run it again with the
-same named, cpud_test, you must:
-```
-docker rm cpud_test
+# If you ran docker before and it failed in some way, you may need to remove the
+# old identity (e.g. docker rm cpud_test)
+docker run --rm -v $KEY.pub,target=/key.pub -v $KEY,target=/key -v /tmp:/tmp --name cpud_test --privileged=true -t -i -p 17010:17010 ghcr.io/u-root/cpu:main
 ```
 
 Then you can try running a command or two:
 ```
-docker exec -it  cpud_test  /bin/cpu -key /key localhost /bin/date
+docker exec -it -e PWD=/ cpud_test  /bin/cpu -key /key localhost /bin/date
```

Comment:
why do I need the PWD=/ ? I never needed to do that before.

Reply:
This is something I experienced on mac when using the docker container to run the client.  PWD is probably set to /Users/erivan01/src/cpu at this point, which doesn't exist inside the docker - and for some reason the fact it can't get to PWD is treated as fatal.

```
(base) FV7GG9FTHL:cpu erivan01$ docker exec -e PWD=/ decpud /bin/decpu $DEBUG -key /key . /bin/date

Sun Oct  9 21:30:10 UTC 2022
(base) FV7GG9FTHL:cpu erivan01$ docker exec decpud /bin/decpu $DEBUG -key /key . /bin/date

2022/10/09 21:30:19 CPUD: your $PWD  is not in the remote namespace
2022/10/09 21:30:19 CPUD(as remote):file does not exist
2022/10/09 21:30:19 Wait: Process exited with status 1
(base) FV7GG9FTHL:cpu erivan01$ 
```

Reply:
PWD is not one of the environment variables set by docker by default: (at least not on OSX Docker Desktop):

(base) FV7GG9FTHL:cpu erivan01$ docker exec -i -t cpud /bin/printenv
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HOSTNAME=589b5cf79160
TERM=xterm
HOME=/

(base) FV7GG9FTHL:cpu erivan01$ docker exec -i -t -e PWD cpud /bin/printenv
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HOSTNAME=589b5cf79160
TERM=xterm
PWD=/Users/erivan01/src/cpu
HOME=/

I'll fix the tests appropriately -- but there is a meta issue obvious here about needing to do something different on OSX (/Users isn't in the docker namespace)

Reply:
I guess part of the way we can deal with this is not make lack of $PWD in namespace fatal?  I mean it might be, but it also might not be, so we should try to run regardless, right?  Although I suppose if we are doing something nasty (rm -rf) then maybe we only want to do that in the right $PWD.  hmmmm.....  the other thing to do is just fix default namespace on OSX

Reply:
so part of the problem here is when running cpu inside the container like the examples from the original documentation -- you are never going to have the paths from the original system (unless you are in /bin, /lib, /lib64 or /root).  We could include instructions for mapping in /home, but what if you are elsewhere?  Its almost like you have to volume mount / to /mnt/local, bind the cpu/cpud binaries to /mnt/local/bbin and then /chroot /mnt/local before executing cpu.  @rminnich - what am I missing?

Reply:
Okay, updated the docs a bit with an explanation why things are different when running the cpu client inside the docker container and added some notes about what to expect when running on OSX.